contents/data_vis_aog_custom.md

```diff
@@ -0,0 +1,46 @@
+## Plot Customizations {#sec:aog_custom}
+
+Apart from mappings inside a `mapping` layer,
+you can **customize `AlgebraOfGraphics` visualizations inside the `visual` transformation layers**.
+
+For example the `linear` statistical transformation plot from @sec:aog_stats can be customized both with the marker objects in the scatter plot but also with the line object in the linear trend plot.
+We can customize anything that the `Makie.jl`'s plotting types support inside `visual`:
+
+```jl
+s = """
+    # aog_custom_visual # hide
+    CairoMakie.activate!() # hide
+    plt = data(synthetic_df) *
+        mapping(:x, :y) *
+        (
+            visual(Scatter; color=:steelblue, marker=:cross)
+            + (
+                linear() * visual(; color=:red, linestyle=:dot, linewidth=5)
+            )
+        )
```

Comment:
```suggestion
    blue = visual(Scatter; color=:steelblue, marker=:cross)
    red = linear() * visual(; color=:red, linestyle=:dot, linewidth=5)
    plt = data(synthetic_df) * mapping(:x, :y) * (blue + red)
```

I suggest to split it up here to make the structure of the full equation more readable. One-liners are easier to build, but less readable generally.

Also, original code had inconsistent infix operators location: `*` is at the end of line 14 and `+` is at the start of line 17.

Reply:
Thanks!